pocs/tests/test_scheduler.py

```diff
@@ -22,7 +22,7 @@ def test_bad_scheduler_fields_file(config):
 
 
 def test_no_observer(config):
-    assert create_scheduler_from_config(config, observer=None) is None
+    create_scheduler_from_config(config, observer=None)
```

Comment:
Without an `assert` statement there isn't actually any test performed. Probably want to check the return type is actually a scheduler.